packages/react-devtools-extensions/src/panel.js

```diff
@@ -17,3 +17,32 @@ window.injectStyles = getLinkTags => {
     }
   }
 };
+
+// Remove the banner if the user has closed the banner before
+// eslint-disable-next-line no-undef
+chrome.storage.local.get(['bannerClosed']).then(result => {
+  if (result.bannerClosed) {
+    const banner = document.querySelector('.dismissable-banner');
+    banner.remove();
+  }
+});
```

Comment:
Since `storage.local.get` is asynchronous, this will probably leave the banner on the screen for a couple of milliseconds?

Can we actually do the opposite thing here? Instead of hiding based on the flag from storage, can we show the banner, if there is no `bannerClosed` flag in storage?